web/server/static/static-threaded.c

```diff
@@ -454,6 +455,19 @@ static void socket_listen_main_static_threaded_cleanup(void *ptr) {
     static_thread->enabled = NETDATA_MAIN_THREAD_EXITED;
 }
 
+static int is_there_react_dashboard() {
+    char webfilename[FILENAME_MAX + 1];
+    snprintfz(webfilename, FILENAME_MAX, "%s/index-new.html", netdata_configured_web_dir);
+
+    FILE *fs = fopen(webfilename, "r");
+    if(fs) {
+        fclose(fs);
+        return 1;
+    }
+
+    return 0;
+}
+
```

Comment:
This should be removed and replaced with the respective functionality either at the installer level, or some script. There's no way we're going to have a C function that's coupled to the version or implementation of the dashboard.